numpy/add_newdocs.py

```diff
@@ -5160,7 +5164,7 @@ def luf(lamdaexpr, *args, **kwargs):
 
     Returns
     -------
-    out : ndarray of ints
+    i : ndarray of ints
```

Comment:
Hmm, `i` is and index, not an array of indexes. I like `out` or `ret` better.

Reply:
Maybe `indices`? (as for `searchsorted`)

Reply:
Reasoning was that `v` is/was used in the comparison expressions to mean a single value, despite being the argument name for an array of values, For consistency then, I'm using `i` in those expressions to mean a single index, as well as the returned array of values.